contracts/Kiftables.sol

```diff
@@ -123,15 +136,24 @@ contract KiftVans is ERC721, IERC2981, Ownable, ReentrancyGuard, VRFConsumerBase
 
     // ============ Airdrop ============
 
-    function airdropMint() public onlyOwner {
-        for (uint256 i = 0; i < maxAirdroppedVans; i++) {
-            // using _mint saves $5 ($386 vs $391)
-            // TODO use IERC721Receiver to support address(this) instead of msg.sender
-            _safeMint(msg.sender, nextTokenId());
-        }
+    // this will mint 1000 tokens to the contract
+    // these can be transferred to contributors etc
+    function treasuryMint() public onlyOwner {
+        require(treasuryMinted == false, "Treasury can only be minted once");
+
+        _safeMint(msg.sender, maxTreasuryKiftables);
+
+        // for (uint256 i = 0; i < maxTreasuryKiftables; i++) {
+        // TODO decide on _mint vs _safeMint - needs gas testrun
+        // TODO use IERC721Receiver to support address(this) instead of msg.sender
+        // _mint(msg.sender, nextTokenId());
+        // }
+
+        treasuryMinted = true;
     }
 
-    function airdropTransfer(address _to, uint256[] memory _tokenIds)
+    // TODO should this just take a count to be transferred and be random?
+    function bulkTransfer(address _to, uint256[] memory _tokenIds)
```

Comment:
this is unused except for a single test. what do you need it for? consider removing?